services/scabbard/libscabbard/src/service/mod.rs

```diff
@@ -67,7 +67,7 @@ const SERVICE_TYPE: &str = "scabbard";
 const DEFAULT_COORDINATOR_TIMEOUT: u64 = 30; // 30 seconds
 
 /// Specifies the version of scabbard to use.
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, PartialEq)]
```

Comment:
Is this derive necessary? If it's only for tests, we might consider using something like

```rust
#[cfg_attr(test, derive(PartialEq))]
```

Reply:
its required by this check https://github.com/Cargill/splinter/pull/1306/files#diff-8407c21da2f6e6d430d3275579d5723b307d0dab6cb18fda50c9edee59cf9be6R139